File: vn/src/main/java/org/sqlbuilder/vn/VnSensekeyResolver.java
package org.sqlbuilder.vn;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;
import java.util.AbstractMap.SimpleEntry;

public class VnSensekeyResolver extends Resolver<String, SimpleEntry<Integer, Integer>>
{
	public VnSensekeyResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: vn/src/main/java/org/sqlbuilder/vn/VnSensekeyResolver.kt
package org.sqlbuilder.vn

import org.sqlbuilder.common.DeSerialize.deserialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias VnSensekeyResolvable = String?
typealias VnSensekeyResolved = Pair<Int, Int>

class VnSensekeyResolver(ser: String) : Resolver<VnSensekeyResolvable, VnSensekeyResolved>(deserialize<Map<VnSensekeyResolvable, VnSensekeyResolved>>(File(ser)))
